src/Perspex.Controls/Expander.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Perspex.Animation;
+using Perspex.Controls.Primitives;
+
+namespace Perspex.Controls
+{
+    public enum ExpandDirection
+    {
+        Down,
+        Up,
+        Left,
+        Right
+    }
+
+    public class Expander : HeaderedContentControl
+    {
+        static Expander()
+        {
+            PseudoClass(ExpandDirectionProperty, d => d == ExpandDirection.Down, ":ExpandDirectionDown");
```

Comment:
I think pseudoclasses should be kept short, and all other pseudoclasses are lowercase-only, so probably ":down", ":up" etc would suffice. However, it seems you're using `Expander[ExpandDirection=Down]` in the style so are these even necessary?